### ISSUE TITLE
Colored emboss filter?

### ISSUE THREAD

**Issue by gapipro** [pixijs/pixi-filters#166]
Currently it looks like emboss filter is only grayscale. Is it possible to make it colored? Similar to emboss effect in Photoshop.

**Comment by bigtimebuddy** [comment]
No, sorry. It's only gray-scale. If someone wants to support a tint color, PR welcomed.
